css/properties/font-synthesis-small-caps.json

```diff
@@ -0,0 +1,58 @@
+{
+  "css": {
+    "properties": {
+      "font-synthesis-small-caps": {
+        "__compat": {
+          "spec_url": "https://drafts.csswg.org/css-fonts/#font-synthesis-small-caps",
+          "support": {
+            "chrome": {
+              "version_added": "97"
+            },
+            "chrome_android": {
+              "version_added": "97"
+            },
+            "edge": {
+              "version_added": false
```

Comment:
The other Chromium-based browsers are all but certain to ship this feature at the same time. It'd be nice to see that reflected in the data here.

You can find information on the map from Chrome (Blink) to other browsers in the browsers data: https://github.com/mdn/browser-compat-data/tree/main/browsers. For example, Edge (and WebView) would be 97 and Opera would be 83. If the browsers data doesn't map to 97 or later, then leave it as false (e.g., Opera Android). It's OK that some of the releases are in the future, provided it's consistent with the `browsers` data.

This would apply throughout the PR, for Edge, Opera, and WebView Android.

Reply:
Sorry for the late reply and thank you for your comment, @ddbeck! 
Following your suggestions, I have changed versions for other Chromium-based browsers as well.